src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py

```diff
@@ -717,3 +731,276 @@ def _test_replica_mgmt(self, resource_group, skuName, testSkuName, isBasicTier):
+            'subnet': self.create_random_name('cli-subnet-', 24),
+            'pe': self.create_random_name('cli-pe-', 24),
+            'pe_connection': self.create_random_name('cli-pec-', 24)
+        })
+
+        # Prepare server and network
+        result = self.cmd('mariadb server create -g {} --name {} -l brazilsouth '
+                          '--admin-user cloudsa --admin-password SecretPassword123 '
+                          '--sku-name GP_Gen5_2'
+                          .format(resource_group, self.kwargs['server_name']),
+                          checks=[
+                              JMESPathCheck('name', self.kwargs['server_name']),
+                              JMESPathCheck('resourceGroup', resource_group),
+                              JMESPathCheck('sslEnforcement', 'Enabled'),
+                              JMESPathCheck('sku.name', 'GP_Gen5_2')]).get_output_in_json()
+
+        sleep(300)
+
+        self.kwargs['mariadb_id'] = result['id']
+        self.cmd('network vnet create -n {vnet} -g {rg} -l {loc} --subnet-name {subnet}',
+                 checks=self.check('length(newVNet.subnets)', 1))
+        self.cmd('network vnet subnet update -n {subnet} --vnet-name {vnet} -g {rg} '
+                 '--disable-private-endpoint-network-policies true',
+                 checks=self.check('privateEndpointNetworkPolicies', 'Disabled'))
+
+        # Create a private endpoint connection
+        pe = self.cmd('network private-endpoint create -g {rg} -n {pe} --vnet-name {vnet} --subnet {subnet} -l {loc} '
+                      '--connection-name {pe_connection} --private-connection-resource-id {mariadb_id} '
+                      '--group-ids server').get_output_in_json()
+        self.kwargs['pe_id'] = pe['id']
+
+        # Show the connection at server side
+        mariadb = self.cmd('mariadb server show -n {mariadb}',
+                           checks=self.check('length(properties.privateEndpointConnections)', 1)).get_output_in_json()
+        self.kwargs['mariadb_pec_id'] = mariadb['properties']['privateEndpointConnections'][0]['id']
+        self.cmd('mariadb server private-endpoint-connection show --id {mariadb_pec_id}',
+                 checks=self.check('id', '{mariadb_pec_id}'))
+        self.kwargs['mariadb_pec_name'] = self.kwargs['mariadb_pec_id'].split('/')[-1]
+        self.cmd('mariadb server private-endpoint-connection show --server-name {server_name} --name {mariadb_pec_name}',
+                 checks=self.check('name', '{mariadb_pec_name}'))
+        self.cmd('mariadb server private-endpoint-connection show --server-name {server_name} -n {mariadb_pec_name}',
+                 checks=self.check('name', '{mariadb_pec_name}'))
+
+        # Test approval/rejection
+        self.kwargs.update({
+            'approval_desc': 'You are approved!',
+            'rejection_desc': 'You are rejected!'
+        })
+        self.cmd('mariadb server private-endpoint-connection reject --id {mariadb_pec_id} '
```

Comment:
It's already a long running operation. There is an error in keyvault sdk. We don't need this one. Meanwhile, please support --no-wait and add `wait` command

Reply:
removed